src/main/scala/firrtl_interpreter/Driver.scala

```diff
@@ -10,7 +10,9 @@ case class InterpreterOptions(
     setOrderedExec:    Boolean              = false,
     allowCycles:       Boolean              = false,
     randomSeed:        Long                 = System.currentTimeMillis(),
-    blackBoxFactories: Seq[BlackBoxFactory] = Seq.empty)
+    blackBoxFactories: Seq[BlackBoxFactory] = Seq.empty,
+    maxExecutionDepth: Long                 = ExpressionExecutionStack.defaultMaxExecutionDepth
+    )
```

Comment:
We are moving to a style guide that says closing paren on line 15 should be part pulled up to the end of line 14.  I kinda of prefer the way you did it. I don't think we have style checking for that yet